app/views/external_users/claims/case_details/_cracked_trial_fields.html.haml

```diff
@@ -1,22 +1,28 @@
-#cracked-trial-dates.form-section{ class: @claim&.case_type&.requires_cracked_dates? ? '' : 'hidden', 'aria-labelledby': 'cracked-trial-dates-heading' }
-  %h3#cracked-trial-dates-heading.govuk-heading-m
+#cracked-trial-dates{ class: @claim&.case_type&.requires_cracked_dates? ? nil : 'hidden' }
+  %h3.govuk-heading-m
     = t('.cracked_trial_details')
 
-  = f.gov_uk_date_field(:trial_fixed_notice_at, legend_text: t('.trial_fixed_notice_at'), legend_class: 'govuk-legend', form_hint_text: t('.date_hint'), id: "trial_fixed_notice_at", error_messages: gov_uk_date_field_error_messages(@error_presenter, :trial_fixed_notice_at))
+  #trial_fixed_notice_at
+    = f.govuk_date_field :trial_fixed_notice_at,
+      maxlength_enabled: true,
+      legend: { text: t('.trial_fixed_notice_at') },
+      hint: { text: t('.date_hint') }
 
```

Comment:
We can get rid of the encapsulating container id `trial_fixed_notice_at` with an added option to the component : `form_group: { id: 'trial_fixed_notice_at' }`. 

[Reference](https://www.rubydoc.info/gems/govuk_design_system_formbuilder/GOVUKDesignSystemFormBuilder/Builder#:~:text=permalink-,%23govuk_date_field(attribute_name%2C%20hint%3A%20%7B%7D%2C%20legend%3A%20%7B%7D%2C%20caption%3A%20%7B%7D%2C%20date_of_birth%3A%20false%2C%20omit_day%3A%20false%2C%20maxlength_enabled%3A%20false%2C%20form_group%3A%20%7B%7D%2C%20wildcards%3A%20false%2C%20**kwargs%2C%20%26block)%20%E2%87%92%20ActiveSupport%3A%3ASafeBuffer,-Note%3A%20When%20using)